python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1249,7 +1249,7 @@ def convert_argmax(node, **kwargs):
     attrs = node["attrs"]
 
     axis = int(attrs.get("axis"))
-    keepdims = int(attrs.get("keepdims")) if "keepdims" in attrs  else 1
+    keepdims = get_boolean_attribute_value(attrs, "keepdims") if "keepdims" in attrs else 1
```

Comment:
this can be just `get_boolean_attribute_value(attrs, "keepdims")` 

and `if "keepdims" in attrs else 1` can be removed, because the "get_boolean_attribute_value" method handles non-existing keys.